.bunnyshell/templates/playwright/bunnyshell.yaml

```diff
@@ -0,0 +1,53 @@
+kind: Environment
+name: playwright-e2e
+type: primary
+urlHandle: jlhjuq
+environmentVariables:
+    SAMPLE_VAR: '{{env.unique}}'
+    VITE_APP_NAME: 'Playwright Test'
+components:
+    -
+        kind: Application
+        name: application
+        gitRepo: 'https://github.com/bunnyshell/template-vue-tailwind.git'
+        gitBranch: main
+        gitApplicationPath: /
+        dockerCompose:
+            build:
+                context: .
+                dockerfile: docker/app/Dockerfile
+                args:
+                    VITE_APP_NAME: '{{env.vars.VITE_APP_NAME}}'
```

Comment:
do we really need this argument? the idea is that the users will not be able to use the prebuilt images for this template, because they will have this always diferent argument